app/views/backend/entities/show.html.haml

```diff
@@ -1,3 +1,8 @@
+- if resource.nature== 'contact'
```

Comment:
`==` should be spaced around.

`resource.nature == 'contact'` instead of current code would be better.